e2e/utils.go

```diff
@@ -1190,6 +1191,93 @@ func k8sVersionGreaterEquals(c kubernetes.Interface, major, minor int) bool {
 	return (v.Major > maj) || (v.Major == maj && v.Minor >= min)
 }
 
+// kernelVersion holds kernel related information.
```

Comment:
currently we have kernel extraction code at internal.utils.go can we reuse it here?

Reply:
Sure, vim was removing when I tried to include it, thought there is some rule not to include the internal/* files in e2e :-)